src/EnergyPlus/OutdoorAirUnit.hh

```diff
@@ -124,15 +159,14 @@ namespace OutdoorAirUnit {
         // Members
         // Equipment List Data
         std::string ComponentName;
-        std::string ComponentType;
```

Comment:
A lot of `std::string` usages like these were replaced with lookups from the `constexpr std::arrays`. But a lot of work still remains.